README.md

```diff
@@ -101,6 +101,12 @@ should be installed as one of your project's `devDependencies`:
 ```
 npm install --save-dev @testing-library/react
 ```
+or
+
+This module is distributed via [yarn](https://classic.yarnpkg.com/en/)
```

Comment:
yarn isn't a distribution channel but a package manager. What do you want to express with that statement?

Reply:
Oh yeah, you are right, I just made a mistake there.